etabotsite/etabotsite/settings.py

```diff
@@ -41,9 +45,49 @@
     logging.warning('cannot load custom_settings.json due to "{}"'.format(
         e))
 
+CUSTOM_SETTINGS = custom_settings
+PROD_HOST_URL = prod_host_url
+
+# Determine if we are running on local mode or production mode
+# This is based on custom settings config.
+# This makes control of prod environment more controlable cross-platform
+LOCAL_MODE = custom_settings.get("LOCAL_MODE")
```

Comment:
can you set default to False please?
so that we don't have to updated production settings

Reply:
Changed it to be an if that checks if Local Mode is in custom settings, then sets it to whatever is in settings if so. If not it will set it to false!